examples/getstarted/config/admin.js

```diff
@@ -6,4 +6,7 @@ module.exports = ({ env }) => ({
   apiToken: {
     salt: env('API_TOKEN_SALT', 'example-salt'),
   },
+  relations: {
+    performance: false,
```

Comment:
IMO performance isn't clear to users, maybe I would do "defaultPopulate" or just "populate"....

Reply:
I agree here. Maybe this could follow the populate semantics we have for the API even? Also as mentioned yesterday I don't think this should live in the admin configuration as it does not configure the admin interface. Webhooks do have their own configuration and imho these options should be set there. Wdyt?

Reply:
If it was set in the `webhook` config, would that mean it's only isolated to the webhook return? Because at the moment it seems this change has the potential to reverse the performance work in the admin panel which imo we shouldn't allow at all?

Reply:
Imho this should be limited to the webhook payload and nothing else for now.

Reply:
🤝🏼 alignment

Reply:
> Webhooks do have their own configuration and imho these options should be set there. Wdyt?

Sounds like it should live there indeed.


> Because at the moment it seems this change has the potential to reverse the performance work in the admin panel which imo we shouldn't allow at all?

I align with that too. The cleanest way to do this I think will be to make another database query with the populate attributes.

It will make the request slower, but I think it's the cleanest way to do it.

Reply:
I tend to agree, this file is to override the admin also adding this configuration on the file system means that it is not configurable at runtime, maybe it could be a setting of the relation field in the CM?

Reply:
@soupette It doesn't affect the admin app at all and instead should only change the webhook payload. I, therefore, believe it shouldn't be connected with the schema or the admin at all. This is mostly to revert a breaking change that we missed.

Reply:
I mildly disagree here, this impacts both webhooks and lifecycles. I assume the fix does too depending on when/where the webhook payload is set.

Reply:
yes, this will definitely impact both webhooks and lifecycles. Both have to receive the populated relations.

I was trying to think of a name for the config variable, which encapsulates both concepts. But I still have not figured out any that feels ok.

Reply:
@Marc-Roig Wouldn't that be another reason to only revert the breaking change and introduce the config later on? It feels like this needs a good definition if it impacts several places.